cli/deployment.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+)
+
+func init() {
+	register("deploy", runDeployments, `
+usage: flynn deploy
+
+Manage app deployments
+
+Commands:
+    With no arguments, shows a list of deployments
+
+Examples:
+
+	$ flynn deploy
```

Comment:
I think we should probably call this `flynn deployment` to avoid the confusion of an action vs a namespace.